docs/user-guide/map-views.md

```diff
@@ -0,0 +1,77 @@
+# Map Views
+
+The **Map Views** is a [MapStore](https://mapstore.geosolutionsgroup.com/mapstore/#/) tool useful to configure multiple views differently configured and switch between. A navigation functionality is also provided to automatically activate each view one after the other in temporal sequence.
+
+!!! note
+    The *Map views* works in both 2D and 3D modes, but the 3D mode has advanced options including the *Mask*, the *Globe Translucency* and the *Clipping* (see next paragraph).
+
+## Add new view
+
+Once the user opens a map, the *Map Views* tools can be open through the <img src="../img/button/map-views-button.jpg" class="ms-docbutton" /> button available in the [Side Toolbar](mapstore-toolbars.md#side-toolbar).
+
+<img src="../img/map-views/map-views-panel.jpg" class="ms-docimage"/>
+
+To create a new simple view the user can simply move the map to the interested area, enable the desired layers in TOC to be displayed in the map and finally click on the <img src="../img/button/+++.jpg" class="ms-docbutton" /> button. The view will be created and visible in the *Views* list by clicking the <img src="../img/button/timeline-layers-list-button.jpg" class="ms-docbutton" /> button.
+
+<img src="../img/map-views/add-view.gif" class="ms-docimage"/>
+
+[MapStore](https://mapstore.geosolutionsgroup.com/mapstore/#/) allows to customize and edit the new view by opening the Edit panel through the <img src="../img/button/editing-button.jpg" class="ms-docbutton" /> button. Here the user is allowed to:
+
+* Add a text, images, videos or hyperlinks by clicking **Description** section. The *description* is visible during the [Views Navigation](map-views.md#3d-views-navigations). Take a look at the following example.
+
+<img src="../img/map-views/description.jpg" class="ms-docimage"/>
+
+* *Capture the view positions* through the <img src="../img/button/capture-view-position-button.jpg" class="ms-docbutton" /> button or change the longitude, latitude and height of the *Camera Position* and *Center Position*  (only available for the *3D* mode) by clicking on **Position** section.
+
+<img src="../img/map-views/position.jpg" class="ms-docimage"/>
+
+* Modify the duration of the animation and enable/disable the transition during the [Views Navigation](map-views.md#3d-views-navigations) by clicking on **Animation** section.
+
+<img src="../img/map-views/animation.jpg" class="ms-docimage"/>
+
+* Select a WFS or Vector layer available in the map to create a mask on all the 3D tiles visible in the map by clicking **Mask** section. The WFS or Vector layer is added to the map by the user and then used the layers features in a transparent way to create the clipping area.
+
+<img src="../img/map-views/mask-panel.jpg" class="ms-docimage"/>
+
+If the user enables the **Inverse** option, a hole is created inside the 3D Tile layers. This hole can be filled by only 3D objects (3D Tiles cannot be used to fill these holes because all 3D tiles will be masked).
```

Comment:
If the user enables the **Inverse** option, simply the inverse mask is applied using the same layer so that each feature is used to produce an hole on all visible 3D Tiles.